book/src/advanced_database_migrations.md

```diff
@@ -16,6 +16,7 @@ validator client or the slasher**.
 
 | Lighthouse version | Release date | Schema version | Downgrade available? |
 |--------------------|--------------|----------------|----------------------|
+| v7.1.0             | TBD 2025     | v23            | yes                  |
```

Comment:
I've added a placeholder version here, we'll need to update to include v7.0.0 as well

Reply:
Nice. 

v7.0.0 will be introduced here 

- https://github.com/sigp/lighthouse/pull/7280